packages/botonic-react/src/components/message.jsx

```diff
@@ -154,31 +169,66 @@ export const Message = props => {
 
   const isFromUser = () => from === 'user'
   const isFromBot = () => from === 'bot'
+
   const getBgColor = () => {
     if (!blob) return COLORS.TRANSPARENT
-    return isFromUser()
-      ? getThemeProperty('brand.color', COLORS.BOTONIC_BLUE)
-      : COLORS.SEASHELL_WHITE
+    if (isFromUser()) {
+      const color = getThemeProperty('message.user.style.background', undefined)
+      return color
+        ? color
+        : getThemeProperty('brand.color', COLORS.BOTONIC_BLUE)
+    }
+    const color = getThemeProperty('message.bot.style.background', undefined)
+    return color ? color : COLORS.SEASHELL_WHITE
   }
 
   const getMessageStyle = () =>
     isFromBot()
       ? getThemeProperty('message.bot.style')
       : getThemeProperty('message.user.style')
 
-  const getBlobTick = () => getThemeProperty(`message.${from}.blobTick`, true)
+  const hasBlobTick = () => getThemeProperty(`message.${from}.blobTick`, true)
 
   const renderBrowser = () => {
     const m = webchatState.messagesJSON.find(m => m.id === state.id)
     if (!m || !m.display) return <></>
-    const pointerSize = 6
-    const pointerStyles = {
-      position: 'absolute',
-      top: '50%',
-      width: 0,
-      height: 0,
-      border: `${pointerSize}px solid ${COLORS.TRANSPARENT}`,
-      marginTop: -pointerSize,
+
+    const getBlobTick = pointerSize => {
+      const color =
+        pointerSize == 5
+          ? getBgColor()
+          : getThemeProperty(
+              `message.${from}.style.borderColor`,
+              COLORS.TRANSPARENT
+            )
+      const containerUserStyle = {
+        ...getThemeProperty('message.user.blobTickStyle'),
+        right: 0,
+        marginRight: -pointerSize,
+      }
+      const blobTickUserStyle = {
+        borderLeftColor: color,
+        borderRight: 0,
+      }
+      const containerBotStyle = {
+        ...getThemeProperty('message.bot.blobTickStyle'),
+        left: 0,
+        marginLeft: -pointerSize,
+      }
+      const blobTickBotStyle = {
+        borderRightColor: color,
+        borderLeft: 0,
+      }
```

Comment:
It looks good the way you put it together. You defined first the `user` styles and then the `bot` styles. There's a tiny improvement if you want, not a must.
You have `4` variables here and only `2` of them will be used at the same time. Probably I'd go for

```
const containerStyle = {
  ...getThemeProperty('message.user.blobTickStyle'),
}
const blobTickStyle = {}

if (isFromUser()) {
  containerStyle.left: 0
  containerStyle.marginLeft: -pointerSize
  blobTickStyle:borderLeftColor: color
} else {
  containerStyle.right: 0
  containerStyle.marginRight: -pointerSize
  blobTickStyle:borderRightColor: color
}

... 

return (
  <BlobTickContainer
    style={containerStyle}
  >
    <BlobTick
      pointerSize={pointerSize}
      style={blobTickStyle}
    />
  </BlobTickContainer>
)
```

Again, up to you!

Reply:
this would be a nice improvement! good catch 👍